extensions/wikia/ArticleVideo/bower_components/html5-skin/tests/controller-test.js

```diff
@@ -193,6 +198,7 @@ OO = {
       displayMoreOptionsScreen: function(a) {},
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove the unused function parameter "a". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AUnusedFunctionArgument)